std/src/main/scala/algebra/std/vector.scala

```diff
@@ -0,0 +1,41 @@
+package algebra
+package std
+
+import scala.annotation.tailrec
+
+package object vector extends VectorInstances
+
+trait VectorInstances {
+  implicit def vectorOrder[A: Order]: Order[Vector[A]] = new VectorOrder[A]
+  implicit def vectorMonoid[A]: Monoid[Vector[A]] = new VectorMonoid[A]
+}
+
+final class VectorOrder[A](implicit ev: Order[A]) extends Order[Vector[A]] {
```

Comment:
Sure. I used lexicographic order because I saw that's what the Order[List[A]] implementation was doing. Comparing vector length first would certainly be more efficient. I don't really see myself ever using either, so if others have opinions on what this should be doing, I'm happy to follow them.

This also makes me realize that if we take this approach, I should override `eqv` to first check that the lengths are equal.